pymc_marketing/mmm/plot.py

```diff
@@ -1479,213 +1480,444 @@ def allocated_contribution_by_channel_over_time(
         fig.tight_layout()
         return fig, axes
 
-    def plot_sensitivity_analysis(
+    def sensitivity_analysis(
```

Comment:
Breaking change. Can we provide previous method as a wrapper to this and and warning